src/sass/_theme_layout.sass

```diff
@@ -334,7 +334,7 @@ html
   width: $nav-desktop-width + 20px
   position: relative
   overflow-x: hidden
-  overflow-y: scroll
+  overflow-y: hidden
```

Comment:
`scroll` might be needed? Did you get a chance to test with a long nav menu? I'm not sure exactly where this would be needed prior, but my guess this is going to clip a long navigation menu that is taller than the viewport.

Reply:
I reverted this change because it was not needed.